modules/rostests/apitests/user32/LoadImageGCC.c

```diff
@@ -0,0 +1,256 @@
+            sprintf(buffer, "%p", handle);
+//            MessageBoxA(NULL, buffer, "handle", 0);
+            hBmp = (HBITMAP)LoadImage(handle, MAKEINTRESOURCE(130), IMAGE_BITMAP, 0, 0, LR_DEFAULTCOLOR);
+            sprintf(buffer, "%p", hBmp);
+//            MessageBoxA(NULL, buffer, "Bmp", 0);
+            sprintf(buffer, "%ld", GetLastError());
+//            MessageBoxA(NULL, buffer, "LastError", 0);
+            break;
+        }
+
+        case WM_PAINT:
+        {
+            PAINTSTRUCT ps;
+            HDC hdc, hdcMem;
+            BITMAP bitmap;
+            BITMAPINFO bmi;
+            hdc = BeginPaint(hWnd, &ps);
+            HGLOBAL hMem;
+            LPVOID lpBits;
+            CHAR img[8] = { 0 };
+            UINT size;
+
+            hdcMem = CreateCompatibleDC(hdc);
+            SelectObject(hdcMem, hBmp);
+            GetObject(hBmp, sizeof(BITMAP), &bitmap);
+            sprintf(buffer, "H = %ld, W = %ld", bitmap.bmHeight, bitmap.bmWidth);
+//            MessageBoxA(NULL, buffer, "test", 0);
+            memset(&bmi, 0, sizeof(bmi));
+            bmi.bmiHeader.biSize        = sizeof(BITMAPINFOHEADER);
+            bmi.bmiHeader.biWidth       = bitmap.bmWidth;
+            bmi.bmiHeader.biHeight      = bitmap.bmHeight;
+            bmi.bmiHeader.biPlanes      = bitmap.bmPlanes;
+            bmi.bmiHeader.biBitCount    = bitmap.bmBitsPixel;
+            bmi.bmiHeader.biCompression = BI_RGB;
+            bmi.bmiHeader.biSizeImage   = 0;
+
+            size = ((bitmap.bmWidth * bmi.bmiHeader.biBitCount + 31) / 32) * 4 * bitmap.bmHeight;
+            sprintf(buffer, "size = %d", size);
+//            MessageBoxA(NULL, buffer, "size", 0);
+
+            hMem = GlobalAlloc(GMEM_MOVEABLE, size);
+            lpBits = GlobalLock(hMem);
+            GetDIBits(hdc, hBmp, 0, bitmap.bmHeight, lpBits, &bmi, DIB_RGB_COLORS);
+
+            /* Get bottom line of bitmap */
+            memcpy(img, lpBits, 8);
+            sprintf(buffer, "0x%02x 0x%02x 0x%02x 0x%02x 0x%02x 0x%02x 0x%02x 0x%02x\n",
+              img[0] & 0xff, img[1] & 0xff, img[2] & 0xff, img[3] & 0xff,
+              img[4] & 0xff, img[5] & 0xff, img[6] & 0xff, img[7] & 0xff);
+//            MessageBoxA(NULL, buffer, "chars 1st Line", 0);
+
+            /* Get one line above bottom line of bitmap */
+            memcpy(img, (VOID *)((INT_PTR)lpBits + 4 * bitmap.bmWidth), 8);
+            sprintf(buffer, "0x%02x 0x%02x 0x%02x 0x%02x 0x%02x 0x%02x 0x%02x 0x%02x\n",
+              img[0] & 0xff, img[1] & 0xff, img[2] & 0xff, img[3] & 0xff,
+              img[4] & 0xff, img[5] & 0xff, img[6] & 0xff, img[7] & 0xff);
+//            MessageBoxA(NULL, buffer, "chars 2nd Line", 0);
+
+            ok(img[0] == 0, "Byte 0 Bad\n");
+            ok(img[1] == 0, "Byte 1 Bad\n");
+            ok(img[2] == 0, "Byte 2 Bad\n");
+            ok(img[3] == 0, "Byte 3 Bad\n");
+
+            GlobalUnlock(hMem);
+            GlobalFree(hMem);              
```

Comment:
```suggestion
            GlobalFree(hMem);
```
EOL-whitespace here. Just mentioned as an example. It's best to strip it in all files entirely using the NPP feature.